src/sugar3/graphics/progressicon.py

```diff
@@ -14,86 +14,88 @@
 # License along with this library; if not, write to the
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
+'''
+The progressicon module provide a progress icon; a widget like progress
+bar which shows progress of a task 
+'''
 
 from gi.repository import Gtk
 from sugar3.graphics.icon import get_surface
 from sugar3.graphics import style
-
-
-class ProgressIcon(Gtk.DrawingArea):
-    """Display the progress filling the icon.
+ 
+class ProgressIcon(Gtk.DrawingArea): '''UI interface for Progress Icon.'''
+    '''Display the progress filling the icon.
 
     This class is compatible with the sugar3.graphics.icon.Icon class.
 
     Call update(progress) with the new progress to update the icon.
-
     The direction defaults to 'vertical', in which case the icon is
     filled from bottom to top.  If direction is set to 'horizontal',
     it will be filled from right to left or from left to right,
     depending on the system's language RTL setting.
 
-    """
-    def __init__(self, icon_name, pixel_size, stroke_color, fill_color,
-                 direction='vertical'):
-        Gtk.DrawingArea.__init__(self)
+    '''
+    def __init__(self, icon_name, pixel_size, stroke_color, fill_color, '''initialize the progressicon '''
```

Comment:
Please do not document here i.e alongside the function parameters. In fact, there is no need for this comment at all.